.github/workflows/reusable_publish_packages.yaml

```diff
@@ -47,31 +47,31 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: falco-${{ inputs.version }}-x86_64.rpm
-          path: /tmp
+          path: /tmp/falco-rpm
```

Comment:
according to the documentation ( https://github.com/actions/download-artifact#compatibility-between-v1-and-v2v3 ) the directory should be created if it doesn't exist.